lib/lead_router/client.rb

```diff
@@ -9,9 +9,10 @@ def initialize(host, user, token)
       @token = token
     end
 
-    def create_lead(site_uuid, lead)
+    def create_lead(site_uuid, lead, destination=nil)
       require_arg "site_uuid", site_uuid
-      request :post, "https://#{@host}/rest/sites/#{site_uuid}/leads", lead.to_json
+      dest_headers = destination ? {"X-ROUTER-DESTINATIONS" => destination} : nil
+      request :post, "https://#{@host}/rest/sites/#{site_uuid}/leads", lead.to_json, dest_headers
     end
```

Comment:
`X-Router-Destinations` is a list of destinations separated by comma `,`:

```
X-Router-Destinations: mojo,foo,bar
```

maybe receive `destinations=[]` as argument and just `.join(",")` then set if > 0